build.sbt

```diff
@@ -69,6 +70,10 @@ val config = Seq(
   "dev.zio" %% "zio-config-magnolia" % zioConfigVersion
 )
 
+val circeYaml = Seq(
```

Comment:
I think you'll need to add this to the projects' dependencies below to the right place.

Interesting that it still compiles - maybe this is a transitive dependency already? It's still a good idea to explicitly define it.